spicy/toolchain/src/compiler/validator.cc

```diff
@@ -1124,6 +1177,63 @@ struct VisitorPost : visitor::PreOrder, hilti::validator::VisitorMixIn {
         if ( auto x = n->op0()->type()->type()->tryAs<type::Unit>(); x && ! x->isFilter() )
             error("unit type cannot be a filter, %filter missing", n);
     }
+
+    static inline std::unordered_map<node::Tag, std::unordered_set<hilti::Attribute::Kind>> allowed_attrs{
```

Comment:
Once more, move to top. Then we also have all tables together, which is nice of future changes/maintainance.

Reply:
Also then add few comments to the tables describing their meaning. 

Basically, my goal would be to have single location to go to and change when we modify/add attributes, so that one doesn't need look around any further (unless more context-sensitivity is needed obviously)

Reply:
I moved the map to the top for both `validator.cc` files, and combined the type attributes in.

I don't want to combine the hilti/spicy tables since they serve a bit different functions, then there isn't a really good place to put spicy attribute mappings that would also be used by the hilti validator. Does one table at the top for hilti and one for spicy seem ok? or is there somewhere else that would make more sense?